esphome/components/esp32_can/esp32_can.h

```diff
@@ -12,6 +12,8 @@ class ESP32Can : public canbus::Canbus {
  public:
   void set_rx(int rx) { rx_ = rx; }
   void set_tx(int tx) { tx_ = tx; }
+  void set_tx_queue_len(uint32_t tx_queue_len) { tx_queue_len_ = tx_queue_len; }
```

Comment:
> All uses of class members and member functions should be prefixed with this-> to distinguish them from global functions/variables.
https://esphome.io/guides/contributing#codebase-standards

```suggestion
  void set_tx_queue_len(uint32_t tx_queue_len) { this->tx_queue_len_ = tx_queue_len; }
```